<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTable</class>
 <widget class="QWidget" name="FormTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonLeft">
       <property name="text">
        <string>Left</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRight">
       <property name="text">
        <string>Right</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Right</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUpdate">
       <property name="text">
        <string>Update</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+U</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBlank">
       <property name="text">
        <string>Blank</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+B</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxLevel"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxType"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRank">
       <property name="text">
        <string>Rank</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOpen">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+O</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGraphicsView" name="graphicsViewPreview"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
